tests/test_it.py

```diff
@@ -69,6 +69,40 @@ def test_b():
     out.assert_outcomes(passed=2, failed=0)
 
 
+def test_it_reuses_the_prev_run_seed(ourtestdir):
+    """
+    Run a test that exercises the randomly-repeat-last option.
+    """
+    ourtestdir.makepyfile(
+        test_one="""
+        def test_a():
+            pass
+        """
+    )
+    out = ourtestdir.runpytest('--randomly-seed=33')
+    out.assert_outcomes(passed=1, failed=0)
+    out.stdout.fnmatch_lines(['Using --randomly-seed=33'])
+    ourtestdir.makepyfile(
```

Comment:
`makepyfile` doesn't need re-running between `runpytest` runs, as the files are left in place